omero/sysadmins/unix/server-installation.txt

```diff
@@ -248,22 +248,23 @@ Ice
 .. note::
 
     OMERO |version| supports 3.5 and 3.4 on UNIX and UNIX-like
-    platforms. You must install the correct version of OMERO.server
-    (see :downloads:`Downloads <>`).
+    platforms; support for 3.6 is currently upcoming. You must install
+    the correct version of OMERO.server (see :downloads:`Downloads
+    <>`).
 
 If possible, install one of the following packages:
 
-+-----------+---------------------------------+
-| System    | Package                         |
-+===========+=================================+
-| BSD Ports | devel/ice                       |
-+-----------+---------------------------------+
-| Debian    | zeroc-ice                       |
-+-----------+---------------------------------+
-| Homebrew  | ice (with --with-python option) |
-+-----------+---------------------------------+
-| RedHat    | N/A (install ZeroC RPMs)        |
-+-----------+---------------------------------+
++-----------+-------------------------------+
+| System    | Package                       |
++===========+===============================+
+| BSD Ports | devel/ice                     |
++-----------+-------------------------------+
+| Debian    | zeroc-ice                     |
++-----------+-------------------------------+
+| Homebrew  | ice (with --with-java option) |
```

Comment:
This is documenting the current state of upstream homebrew.  The `ome` tap is what the separate homebrew page is about, and is deliberately omitted here.  So long as we've adjusted our omero formula in the tap to use the ice from the tap also, that should be covered with no documentation changes required.